File: Data Structures & Algorithms/is-anagram/submission-2.java
class Solution {
    public boolean isAnagram(String s, String t) {
        char [] s_c = s.toCharArray();
        char [] t_c = t.toCharArray();

        Arrays.sort(s_c);
        Arrays.sort(t_c);

        if (s_c.length != t_c.length) return false;

        for (int i = 0; i < s_c.length; i++)
        {
            if (s_c[i] != t_c[i]) {
                return false;
            }
        }

        return true;
    }
}

File: Data Structures & Algorithms/meeting-schedule/submission-2.py
"""
Definition of Interval:
class Interval(object):
    def __init__(self, start, end):
        self.start = start
        self.end = end
"""

class Solution:
    def canAttendMeetings(self, intervals: List[Interval]) -> bool:
        if not intervals:
            return True

        intervals.sort(key=lambda x:x.start)
        n = len(intervals)
        prevEnd = intervals[0]

        for i in range(1, n):
            if prevEnd.end > intervals[i].start:
                return False
            else:
                prevEnd = intervals[i]
        
        return True